emrun.py

```diff
@@ -329,6 +329,10 @@ def create_emrun_safe_firefox_profile():
 user_pref("javascript.options.wasm", true);
 // Enable SharedArrayBuffer (this profile is for a testing environment, so Spectre/Meltdown don't apply)
 user_pref("javascript.options.shared_memory", true);
+// Enable COOP, COEP and CORP headers
+// (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer/Planned_changes)
+user_pref("dom.postMessage.sharedArrayBuffer.withCOOP_COEP", true);
+user_pref("browser.tabs.remote.useCORP", true);
```

Comment:
I don't think these are needed (in Nightly) since this bug has been closed: https://bugzilla.mozilla.org/show_bug.cgi?id=1594748

Reply:
and it's not enabled in FF 71 and will be in 72 right? Can I add a section in emrun documentation ?

Reply:
It'll be 73 I suspect, it landed too late for 72. I'll update MDN.

Reply:
@VirtualTim I rebased last night with the requested changes